EPRN.Portal/Areas/Exporter/Controllers/PRNSController.cs

```diff
@@ -231,6 +226,43 @@ public async Task<IActionResult> DecemberWaste(DecemberWasteViewModel decemberWa
                                     new { decemberWaste.Id });
         }
 
+        [HttpGet]
+        public async Task<IActionResult> DestinationPrn(int? id)
+        {
+            if (id == null)
+                return NotFound();
+
+            var viewModel = await _prnService.GetActionPrnViewModel(id.Value);
+            return View(viewModel);
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> DestinationPrn(DestinationPrnViewModel destinationPrnViewModel)
+        {
+            if (!ModelState.IsValid)
+                return View(destinationPrnViewModel);
+
+            if (destinationPrnViewModel.DoWithPRN == PrnStatus.Draft)
+            {
+                await _prnService.SaveDraftPrn(destinationPrnViewModel);
+                return RedirectToAction(Routes.Areas.Actions.PRNS.PrnSavedAsDraftConfirmation,
+                                new { area = Category.ToString(), destinationPrnViewModel.Id });
```

Comment:
Can we just go through this when you are available? I want to check the operation of this bit.

Reply:
Done.